### ISSUE TITLE
scheduler errors have doubled periods

### ISSUE THREAD

**Issue by jsoref** [kubernetes/kubernetes#123301]
### What happened?

```
0/11 nodes are available: 10 Insufficient memory, 4 Insufficient cpu. preemption: 0/11 nodes are available: 11 No preemption victims found for incoming pod..
```

### What did you expect to happen?

If something is a sentence, it should have only one `.` at the end. If it's an ellipsis, it should have 3 (`...`) or 4 (`....`). If something is a git range it could have 2 (`..`) or 3 (`...`). But a sentence shouldn't have 2 (`..`).

### How can we reproduce it (as minimally and precisely as possible)?

1. Create a number of PriorityClass objects -- we have:

```
NAME                      VALUE        GLOBAL-DEFAULT   AGE
priority01                1000         false            2y18d
priority02                999          false            2y18d
priority03                998          false            2y18d
priority04                997          false            2y18d
priority05                996          false            2y18d
system-cluster-critical   2000000000   false            3y334d
system-node-critical      2000001000   false            3y334d
```

2. Set up a node pool with a limited number of nodes (3)
3. Mostly fill the node pool with pods such that there isn't _much_ space left
4. Create a deployment/replicaset with an HPA(v2) at a low priority (say `priority05`)
5. Have a cronjob that runs at a higher priority (say `priority03`)
6. Investigate pod status and run across some unhappy pods that have `..` in a status

### Kubernetes version

<details>

```console
$ kubectl version
Client Version: v1.28.3
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.28.3-gke.1286000
```

</details>

### Cloud provider

<details>
GKE
</details>

**Comment by kerthcet** [comment]
This has been solved in 1.29 https://github.com/kubernetes/kubernetes/pull/117588.

**Comment by vaibhav2107** [comment]
As per the [comment ](https://github.com/kubernetes/kubernetes/issues/123301#issuecomment-1950975908)the issue is resolved in the PR #117588 and hence can be closed.
